src/components/forms/SignInForm.js

```diff
@@ -3,25 +3,29 @@ import LocalApi from "../../apis/LocalApi";
 import { connect } from "react-redux";
 import { setAuthToken } from "../../actions";
 import { Form, Button, Container } from "react-bootstrap";
+import { Link } from "react-router-dom";
 
 class SigninForm extends Component {
   state = {
     username: "",
-    password: ""
+    password: "",
+    error: null
   };
 
   onFormSubmit = async event => {
     event.preventDefault();
+    const { setAuthToken } = this.props;
```

Comment:
Desctructred 'setAuthToken' to clean code. (We destructured setAuthToken in 'resister form', so I tried to make the code more consistent. )